tests/base_test_mixins.py

```diff
@@ -70,7 +70,7 @@
             "hidden": False,
             "listed": True,
             "accepts_enrollment": True,
-            "git_source": "git://github.com/inducer/relate-sample",
+            "git_source": "git://github.com/dzhuang/relate-sample",
```

Comment:
This should be changed back before merge.

Reply:
Sure.